### PR TITLE
MINOR: Added some tips for running a single test file, test class and/or test method

### PR DESCRIPTION
**Pull request by ppatierno**
Added some tips for running a single test file, test class and/or test method on the documentation landing page about tests

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6344/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6329/
Test PASSed (JDK 8 and Scala 2.12).

File: tests/README.md [hunk 1]
```diff
@@ -25,6 +25,19 @@ _DUCKTAPE_OPTIONS="--debug" bash tests/docker/run_tests.sh | tee debug_logs.txt
...
+```
+TC_PATHS="tests/kafkatest/tests/core/transactions_test.py" bash tests/docker/run_tests.sh
+```
+* Run a specific test class
+```
+TC_PATHS="tests/kafkatest/tests/core/transactions_test.py::TransactionsTest" bash tests/docker/run_tests.sh
+```
+* Run a specific test method
+```
+TC_PATHS="tests/kafkatest/tests/core/transactions_test.py::TransactionsTest.test_transactions" bash tests/docker/run_tests.sh
```
Comment (ijuma) [inline tests/README.md]: Maybe we should pick a test that runs reasonably fast for these examples. Maybe `pluggable_test`?

**Comment by ijuma** [comment]
cc @cmccabe

File: tests/README.md (hunk 1 above)
Comment (ppatierno) [reply tests/README.md]: Totally agree ! I found this "lack" of doc while working on #3434 the I live the transaction_test. Just updated to the pluggable_test !

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6570/
Test FAILed (JDK 7 and Scala 2.11).

**Comment by ppatierno** [comment]
Retest this please

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6574/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6559/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by ppatierno** [comment]
@cmccabe any news on that ?

**Comment by ppatierno** [comment]
I think that this PR is really trivial. Any news @cmccabe @ijuma on that ?

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM. I assume you tested that each of the documented cases works?

**Comment by ppatierno** [comment]
Sure ! I tested them before the commit and I have just re-tested them now to be sure ;) They work.

**Comment by ijuma** [comment]
I double-checked them myself and they work indeed. Merging to trunk.

**Pull request closed without merging**